go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -36,6 +36,7 @@ type nodeInfo struct {
 	// the node's Name
 	name string
 	// The list of physical IPs reported by the gatewayconf annotation
+	// TODO: not used, remove
```

Comment:
just remove this comment and rest is lgtm

Reply:
sure, thanks for noticing